test/conformance/enqueue/urEnqueueKernelLaunchAndMemcpyInOrder.cpp

```diff
@@ -265,14 +316,15 @@ TEST_F(urEnqueueKernelLaunchIncrementMultiDeviceTest, Success) {
     constexpr size_t global_offset = 0;
     constexpr size_t n_dimensions = 1;
 
-    std::vector<uur::raii::Event> Events(devices.size() * 2);
+    std::vector<uur::raii::Event> Events(devices.size() * 2 - 1);
```

Comment:
This is going to be fragile and difficult to modify. I get what you are trying to do, but I suggest we try avoiding too match pointer arithmetic.
Can you just create two vectors?